addons/mail/static/src/components/thread_preview/thread_preview.js

```diff
@@ -67,6 +67,11 @@ class ThreadPreview extends Component {
 
     /**
      * Get inline content of the last message of this conversation.
+     * BE CAREFUL, this value is meant to be based on message.body which is
```

Comment:
Aren't the two about preview safe because they use htmlToTextContentInline that is supposedly escaping stuff? Wasn't that the goal of your other PR?

Reply:
Yes indeed, went too quickly